src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -111,15 +111,16 @@ export class MongoDatabaseTreeItem extends AzureParentTreeItem<IMongoTreeRoot> {
 		}
 
 		if (command.name === 'createCollection') {
-			return withProgress(this.createCollection(stripQuotes(command.arguments.join(','))).then(() => JSON.stringify({ 'Created': 'Ok' })), 'Creating collection');
+			// arguments are always string (ie collection name) whereas arguementObjects will be an Object (where as in arguments, it's a JSON string)
```

Comment:
Can you double check this comment? Feels like it has some duplicate info and also you spelled argu_**e**_ment wrong in one place